docs/guide/sass-modern-compiler.md

```diff
@@ -7,3 +7,63 @@ From version `0.17.0`, this module will configure Nuxt to use the new SASS moder
 If the `sass-embedded` dependency is not installed, the module will configure the `modern` compiler for you. In case you get errors, enable the `disableModernSassCompiler` option in the module configuration to fall back to the `legacy` compiler.
 
 Check [Build Performance](https://vuetifyjs.com/en/features/sass-variables/#build-performance) in Vuetify docs for more details.
+
+## Overriding SASS Variables
+
+Vuetify allows for [overriding global and component-level SASS variables](https://vuetifyjs.com/en/features/sass-variables/).  Setting these up requires configuration that is slightly different from the Vuetify
+documentation while using this Nuxt module.
+
+1)  In your styles directory (for this example, we'll use `${workspace}/assets/styles`), create two files - `assets/styles/main.scss` and `assets/styles/settings.scss`
```

Comment:
As this module is nuxt specific, instead of `${workspace}`, can I suggest using `${srcDir}` as it will cover the old way, and Nuxt 4 new way (`app` folder).